pkg/status/status.go

```diff
@@ -157,7 +156,8 @@ func GetCheckStatus(c check.Check, cs *checkstats.Stats) ([]byte, error) {
 
 // GetDCAStatus grabs the status from expvar and puts it into a map
 func GetDCAStatus(verbose bool) (map[string]interface{}, error) {
-	stats, err := getCommonStatus()
+	// inventory is not enabled for the clusteragent/DCA so we pass nil to getCommonStatus
+	stats, err := getCommonStatus(nil)
```

Comment:
I think it will be great to use `Optional[Component]` instead of `nil`. The benefit of using `Optional[Component]` is that you can assume every component is not `nil`. It can be tricky to know if a component can be nil or not.

Reply:
After discussing this we will fix this in another PR to not block the fix.